app/src/main/java/io/awaken/ui/connections/ConnectionViewHolder.kt

```diff
@@ -30,6 +31,7 @@ class ConnectionViewHolder(
     internal val location = itemView.locationLabel
     internal val date = itemView.awokenDateLabel
     internal val status = itemView.statusMarker
+    private val disposables = CompositeDisposable()
```

Comment:
So the reason we use a `CompositeDisposable` is to be able to cancel our long running network requests and prevent leaking resources. For now we can just not use one, but in the future we should move all of the I/O related logic to a `ViewModel`, and call `disposables.clear()` accordingly.